src/index.ts

```diff
@@ -246,6 +251,16 @@ async function jscodemod(
     log
   });
 
+  // This is intentional.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function safeConsoleLog(...args: any[]) {
```

Comment:
I could have shared this function between `index` and `bin`, but it seems too small to be worth the added indirection.